addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -69,6 +69,10 @@ Wysiwyg.include({
      * @override
      */
     start: async function () {
+        // Bind the _onPageClick handler to click event: to close the dropdown if clicked outside.
+        this.el
+            .closest("#wrapwrap")
```

Comment:
Is there a reason to target the closest `#wrapwrap` ? `this.el` isn't  the `#wrapwrap` here?

Reply:
Not a significant reason. Used to be on the safe side, we can remove that.